app/models/audit_log.py

```diff
@@ -0,0 +1,26 @@
+from app.models.base import TableModelMixin
+from sqlmodel import Field
+from uuid import UUID
+from enum import Enum
+
+
+class EventType(str, Enum):
+    case_created = "Case Created"
+    case_updated = "Update"
```

Comment:
What do you think of prefixing the events with their usage. i.e Case Updated i.e User Login?

Reply:
Completely agree, I have updated the events to use this style. If you can suggest any other events we need to log please do so!
We will need to have a discussion at some point agreeing what we should log, this was just a first attempt.